openerp/tools/mail.py

```diff
@@ -596,6 +597,14 @@ def append_content_to_html(html, content, plaintext=True, preserve=False, contai
         return '%s%s' % (html, content)
     return '%s%s%s' % (html[:insert_location], content, html[insert_location:])
 
+# Inspired by http://stackoverflow.com/questions/517923
+def remove_accents(input_str):
```

Comment:
In tools/mail.py Because we are make this function generic and used in mail  as well as in website_forum.